pixels/behaviours/base.py

```diff
@@ -412,21 +412,45 @@ def process_lfp(self):
             print("> Mapping LFP data")
             data = ioutils.read_bin(data_file, num_chans)
 
+            output = self.processed / recording['lfp_processed']
+            if output.exists():
+                continue
+
+            print("> Performing median subtraction across channels for each timepoint")
+            subtracted = signal.median_subtraction(data, axis=1)
+
             print(f"> Downsampling to {self.sample_rate} Hz")
-            data = signal.resample(data, orig_rate, self.sample_rate)
+            downsampled = signal.resample(subtracted, orig_rate, self.sample_rate)
+            sync_chan = downsampled[:, -1]
```

Comment:
Did you mean to pass this to `sync_data`?

Reply:
i kept the passing to `sync_data` bit as it is, did not touch it. But right now i do not use `sync_data`.